check-pro-mode.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs')
+
+let proMode = false
+
+try {
+  fs.accessSync('./.pro-mode')
+  proMode = true
+} catch (e) {
+  // Do nothing
+}
+
+if (proMode) {
+  console.error('\x1b[31mError! .pro-mode file found!')
```

Comment:
Can use our `chalk` package already installed for colors

Reply:
Keeping this approach so that we don't rely on external packages.